<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="playButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="backButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="media-skip-backward-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="simulButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Launch simulation with the programs [F9]</string>
         </property>
         <property name="statusTip">
          <string>Launch simulation with the programs [F9]</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="system-run-symbolic">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="shortcut">
          <string>F9</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="avanceWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Operation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="sceneTab">
       <attribute name="title">
        <string>Simulated scene</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="SVGWidget" name="svgWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="progTab">
       <attribute name="title">
        <string>Programs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="progEdit"/>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="saveAsButton">
             <property name="text">
              <string>Save &amp;As</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Check Synta&amp;x</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen_Scene"/>
    <addaction name="actionNew_Scene"/>
    <addaction name="actionImport_SVG"/>
    <addaction name="actionInitial_velocities"/>
    <addaction name="actionScale"/>
    <addaction name="actionG"/>
    <addaction name="separator"/>
    <addaction name="action_Quit_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp_F1"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPrograms">
    <property name="title">
     <string>P&amp;rograms</string>
    </property>
    <addaction name="actionSave_standalone"/>
    <addaction name="actionOpen"/>
    <addaction name="actionCheck_Synta_x"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuPrograms"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Scene">
   <property name="text">
    <string>&amp;Open Scene ...</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="text">
    <string>&amp;New Scene</string>
   </property>
  </action>
  <action name="actionImport_SVG">
   <property name="text">
    <string>Import S&amp;VG ...</string>
   </property>
  </action>
  <action name="actionInitial_velocities">
   <property name="text">
    <string>&amp;Initial velocities</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>S&amp;cale</string>
   </property>
  </action>
  <action name="actionG">
   <property name="text">
    <string>&amp;G ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About ...</string>
   </property>
  </action>
  <action name="actionHelp_F1">
   <property name="text">
    <string>&amp;Help (F1)</string>
   </property>
  </action>
  <action name="action_Quit_Ctrl_Q">
   <property name="text">
    <string>&amp;Quit (Ctrl-Q)</string>
   </property>
  </action>
  <action name="actionSave_standalone">
   <property name="text">
    <string>&amp;Save standalone</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionCheck_Synta_x">
   <property name="text">
    <string>Check Synta&amp;x</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SVGWidget</class>
   <extends>QWidget</extends>
   <header>.svgwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
